src/devices/lcec_epocat.c

```diff
@@ -552,51 +635,56 @@ static void lcec_epocat_write(struct lcec_slave *slave, long period) {
   // Selezione idx/probe (0=Probe, 1 = index)
   EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 5, 1);
 
-  // Tristate single ended
-  EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 6, 0);
-
-  // Polarity
-  EC_WRITE_BIT(&pd[hal_data->off_POLARITY], 0, 0);
-  EC_WRITE_BIT(&pd[hal_data->off_POLARITY], 1, 0);
-  EC_WRITE_BIT(&pd[hal_data->off_POLARITY], 2, 0);
-  EC_WRITE_BIT(&pd[hal_data->off_POLARITY], 3, 0);
-  EC_WRITE_BIT(&pd[hal_data->off_POLARITY], 4, 0);
-
   // Check for index enable request from HAL
-  if (*(hal_data->enc_00_index_enable)) {
-    // tell hardware to latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 0, 0);
-  } else {
-    // cancel hardware latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 0, 1);
-  }
-  if (*(hal_data->enc_01_index_enable)) {
-    // tell hardware to latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 1, 0);
-  } else {
-    // cancel hardware latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 1, 1);
-  }
-  if (*(hal_data->enc_02_index_enable)) {
-    // tell hardware to latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 2, 0);
-  } else {
-    // cancel hardware latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 2, 1);
-  }
-  if (*(hal_data->enc_03_index_enable)) {
-    // tell hardware to latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 3, 0);
-  } else {
-    // cancel hardware latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 3, 1);
-  }
-  if (*(hal_data->enc_04_index_enable)) {
-    // tell hardware to latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 4, 0);
-  } else {
-    // cancel hardware latch on index
-    EC_WRITE_BIT(&pd[hal_data->off_RESET_FLAG], 4, 1);
+  if (*(hal_data->enc_00_index_enable))
```

Comment:
Some of the whitespace that I was commenting on at the top of the file--I was trying to figure out what changed here when I eventually noticed that the `{` moved to the next line.  If that hadn't happened, then this chunk of the diff would have been much smaller.

Unfortunately, Github's `ignore whitespace` toggle doesn't appear to treat newlines as whitespace.

Reply:
Sorry, my mistake, feel free to arrange the formatting style as You prefer.